lib/audit/catalog_to_moab_existence.rb

```diff
@@ -0,0 +1,30 @@
+require 'druid-tools'
+
+# Catalog to Moab existence check code
+class CatalogToMoabExistence
+
+  # NOTE: this is a bad thing -- HUGE list of objects;  just for now to populate test db
+  # FIXME:  temporarily turning off rubocop
+  # rubocop:disable all
+  def self.whole_db
+    PreservationCopy.all.each do |pc|
+      id = pc.preserved_object.druid
+      version = pc.current_version
+      storage_location = pc.endpoint.storage_location
+      druid = DruidTools::Druid.new(id)
+      object_dir = "#{storage_location}/#{druid.tree.join('/')}"
+
+      moab = Moab::StorageObject.new(id, object_dir)
+      moab_version = moab.current_version_id
+      if version == moab_version
+        p "hurray - #{id} versions match: #{version}"
+      else
+        p "boo - #{id} catalog has #{version} but moab has #{moab_version}"
+      end
+
+      # TODO: update status, timestamps, report errors, log, etc.
+    end
```

Comment:
Can you write some tickets for these TODOs? Or is that captured in another ticket?

Reply:
I believe the work described by this comment is ticket #108;  I only have this much of a stub here because I needed some actual code for an actual test to determine if the load_fixture stuff worked.